examples/ibc/wasm/wasm_ibc_test.go

```diff
@@ -110,48 +110,45 @@ func TestWasmIbc(t *testing.T) {
 	juno1Chain := juno1.(*cosmos.CosmosChain)
 	juno2Chain := juno2.(*cosmos.CosmosChain)
 
-	// Store ibc_reflect_send.wasm contract
-	ibcReflectSendCodeId, err := juno1Chain.StoreContract(
-		ctx, juno1User.KeyName(), "sample_contracts/ibc_reflect_send.wasm")
+	// Store ibc_reflect_send.wasm contract on juno1
+	juno1ContractCodeId, err := juno1Chain.StoreContract(
+		ctx, juno1User.KeyName(), "sample_contracts/cw_ibc_example.wasm")
```

Comment:
can we add a README.md (in sample_contracts) that links to where `sample_contracts/cw_ibc_example.wasm` upstream is? and the commit used for this compile